R/runoffratio.R

```diff
@@ -35,13 +107,17 @@ runoffratio.sim <-
     return(ans)
   }
 
+
+
 runoffratio.ranges <- function() {
   list(
     rrthresh = c(0, 0.2),
     scale = NA
   )
 }
 
+
+#' @export
 absorbScale.hydromad.runoffratio <-
```

Comment:
This one's an S3 method. Does export know that automatically or is there a different annotation required?